docs/pages/build-reference/variables.mdx

```diff
@@ -1,69 +1,96 @@
 ---
-title: Environment variables and secrets
+title: Environment variables and secrets in EAS Build
```

Comment:
```suggestion
title: Environment variables and secrets
```

i think this title is redundant, given it's in the eas build section in the sidebar and the description on the page says eas build

Reply:
This is the good practice when it comes to serchability. We can alter the sidebar title to shorter form using `sidebar_title` frontmatter field, as for example in here:
* https://github.com/expo/expo/blob/75561368f0467ddf41129245b4a5afc9fe3fbf79/docs/pages/build/eas-json.mdx#L2-L3

Reply:
makes sense to use it in title and then leave it out in sidebar title, yeah